src/MSBuild/Microsoft.Build.CommonTypes.xsd

```diff
@@ -4308,6 +4308,22 @@ elementFormDefault="qualified">
         </xs:annotation>
     </xs:element>
 
+    <xs:element name="WapProjPath" type="msb:StringPropertyType" substitutionGroup="msb:Property">
+        <xs:annotation>
+            <xs:documentation>
+                <!-- _locID_text="WapProjPath" _locComment="" -->Windows Application Packaging project-specific: Path to Windows Application Packaging project root folder.
+            </xs:documentation>
+        </xs:annotation>
+    </xs:element>
+
+    <xs:element name="EntryPointProjectUniqueName" type="msb:StringPropertyType" substitutionGroup="msb:Property">
+        <xs:annotation>
+            <xs:documentation>
+                <!-- _locID_text="EntryPointProjectUniqueName" _locComment="" -->Windows Application Packaging project-specific: Relative path to entry point project file
```

Comment:
Nit to match others:
```suggestion
                <!-- _locID_text="EntryPointProjectUniqueName" _locComment="" -->Windows Application Packaging project-specific: Relative path to entry point project file.
```

Reply:
Thanks Rainer. 

It is even better if we can make it for 16.2.

I will create a new PR for 16.2 branch.

Reply:
@rainersigwald, could you help review the new PR in vs16.2? Thanks.
https://github.com/microsoft/msbuild/pull/4444

Reply:
Per  livarcocc's comment, close 4444 and use this PR